services/horizon/internal/ingest/ingestion.go

```diff
@@ -173,13 +180,38 @@ func (ingest *Ingestion) UpdateAccountIDs(tables []TableName) error {
 	return nil
 }
 
+// Return a channel which emits a message when ingections were commited to the database
+func (ingest *Ingestion) getCommitChannel() chan interface{} {
+	return commitPubsub.SubOnce("commit")
```

Comment:
why magic value `"commit"`? const them at the top or something. this is repeated in other places in the pr